generators/ci-cd/templates/github-artifact.yml.ejs

```diff
@@ -40,13 +40,13 @@ jobs:
         include:
         - os: ubuntu-latest
           executable-suffix: ''
-          native-build-args: --verbose -J-Xmx10g
+          native-build-args: <% if (buildToolMaven) { %>--verbose <% } %>-J-Xmx10g
         - os: macos-latest
           executable-suffix: ''
-          native-build-args: --verbose -J-Xmx13g
+          native-build-args: <% if (buildToolMaven) { %>--verbose <% } %>-J-Xmx13g
         - os: windows-2019
           executable-suffix: '.exe'
-          native-build-args: --verbose -J-Xmx10g
+          native-build-args: <% if (buildToolMaven) { %>--verbose <% } %>-J-Xmx10g
```

Comment:
In my experience, `--verbose` isn't really necessary for Maven. The plugins [documentation](https://graalvm.github.io/native-build-tools/latest/maven-plugin.html) doesn't even really explain what it does.